spdx/parsers/jsonyamlxml.py

```diff
@@ -1264,8 +1265,9 @@ def parse_pkg_originator(self, pkg_originator):
         """
         if isinstance(pkg_originator, str):
             if pkg_originator == "NOASSERTION":
-                return self.builder.set_pkg_originator(self.document, NoAssert())
-            entity = self.builder.create_entity(self.document, pkg_originator)
+                return self.builder.set_pkg_originator(self.document,utils.NoAssert())
```

Comment:
Stylistic check: Can you put a space between the comma and "utils" to be consistent on formatting?